### DISCUSSION TITLE
Wireframe - Retroalimentacion

### DISCUSSION THREAD

**Discussion by BrawnyGG** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#127] [Show and tell]
Hola comunidad de xplorers, les quería mostrar mi wireframe, busco un poco de ayuda pues es mi primer wireframe y no sé si está bien realizado jeje, lo hice en la aplicación de Miro y si lo quieren ver a más detalle, les dejo el [link](https://miro.com/app/board/uXjVOKGTIJg=/?invite_link_id=289760141407)

![image](https://user-images.githubusercontent.com/98553492/155721343-ea932671-d935-4c88-b44b-f6f36332f388.png)

![image](https://user-images.githubusercontent.com/98553492/155721380-5ee05b62-f703-46f5-aa05-26d906953f95.png)

**Comment by KevinSalgado** [comment]
Se ve bastante bien, no veo problema.

**Reply by BrawnyGG** [reply]
Muchas gracias!

**Comment by armap99** [comment]
Se ve bastante profesional.

**Reply by BrawnyGG** [reply]
Wow, en verdad lo valoro mucho, gracias:)

**Comment by romarpla** [comment]
Se ve super bueno!
Muchas gracias por compartir

**Reply by BrawnyGG** [reply]
Muchas gracias MC:)

**Comment by RodolfoMorquecho** [comment]
Se ve genial, por cierto tu UI tambien lo hiciste o haras en miro?

**Reply by BrawnyGG** [reply]
Si, también realicé el UI dentro de miro, te lo dejo en el siguiente [link](https://miro.com/app/board/uXjVOJvDVTo=/?invite_link_id=465901329242) ;)

**Comment by JuanSalvador73** [comment]
Excelente wireframe :D

**Reply by BrawnyGG** [reply]
Muchas gracias Juan! Lo aprecio bastante...

**Comment by AxelPanicoB** [comment]
Si alguien tiene duda de como crear uno encontré este vídeo que a mi me sirvió: https://www.youtube.com/watch?v=STlEdjqN4dI

**Comment by Burstniart** [comment]
Excelente trabajo, se ve padrisimo.

**Comment by OmarVenturaP** [comment]
Felicidades compañero, la verdad esta muy bien tu Wireframe, tanto de UX como de UI, la verdad está bastante intuitivo y a la vez muy practico.  Me gustó mucho.
Saludos

**Reply by BrawnyGG** [reply]
Muchas gracias Omar! Lo aprecio demaciado
